### PR TITLE
Fix CMake build when NWM_META=1

### LINKED ISSUE #572: File does not exist .nwm_version
#### Issue ####
utils/CMakeLists.txt references .nwm_version file when NWM_META=1 but this file does not exist. This causes CMake to crash.

https://github.com/NCAR/wrf_hydro_nwm_public/blob/385b7971febb28d48685b1da3d39e27eb89b7156/trunk/NDHMS/utils/CMakeLists.txt#L7

#### Instructions ####
Build WRF-Hydro using CMake and set environment variable NWM_META=1

### PR DESCRIPTION
**Pull request by danrosen25**
TYPE: bug_fix

KEYWORDS: cmake, NWM_META

SOURCE:
Dan Rosen @danrosen25
ESMF/CGD/UCAR

DESCRIPTION OF CHANGES:
Fix cmake reference to missing .nwm_version file for when NWM_META=1. First checks if file exists. If the file does not exist then NWM_VERSION is set to undefined.

ISSUE:
```
Fixes #572
```

TESTS CONDUCTED:
Built on Jet within UFS application with NWM_META=1. Ran coupled UFSATM-WRFHYDRO configuration

### Checklist
 - [X] Closes issue #572 
 - [ ] Tests added (unit tests and/or regression/integration tests)
 - [X] Backwards compatible
 - [ ] Requires new files? If so, how to generate them.
 - [ ] Documentation included
 - [ ] Short description in the Development section of `NEWS.md`

### CODE REVIEW

**Review by kafitzgerald — APPROVED** [review]
Looks good to me.  

Pretty sure this just wasn't getting used for NWM builds so went unnoticed.  

Thanks for the fix!

**Comment by danrosen25** [comment]
@kafitzgerald @rcabell 
I guess we'll need to wait until Cheyenne is up and tests pass to merge this request? I'm currently working on integrating WRFHYDRO into UFS with a NWM build option, which uses CMAKE. I also have some upcoming NUOPC cap modifications to support 3 dimensional lsm soil fields. And while I'm tinkering with the cap I'm going to clean up the code and add a NUOPC test. The test will need configuration files for a small domain. Is this something I should check into the repository? I have an example for a new cap I'm building for a different project. Do you want me to schedule a meeting to show you what I have in mind?

**Review by rcabell — APPROVED** [review]

**Pull request merged**
